### PR TITLE
Update the upload docs

### PR DESCRIPTION
**Pull request by mck182**
Adds some more details to the new upload method
documentation.

### CODE REVIEW

**Comment by gavinsharp** [comment]
Looks great, but we should also move this to a more logical URL and give it a better title (it's no longer about "authentication"). Maybe put it in `basics/upload.md`, and update the reference to it in `README.md` to be clearer?

File: basics/upload.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+name="x-amz-signature"         47aa1...
+name="file"; filename="IMG_20171028_175620-01.jpeg"
+Content-Type: image/jpeg
+
+...the file contents...
+```
+
+***
+
+## Return format
```
Comment (gavinsharp) [inline basics/upload.md]: This part should be removed now I think!

Comment (gavinsharp) [reply basics/upload.md]: Or actually I guess it means the S3 endpoint will return this code. So maybe just rephrase to clarify that?

Comment (mck182) [reply basics/upload.md]: Ah right, I'll rephrase.

File: basics/upload.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Uploading a photo
+To upload a photo, you need to do 2 requests:
+ 1. `POST /photos`
+ 1. `POST url-obtained-from-first-request`
+ 
+First, send a `POST` request to `/photos` ([see documentation](https://github.com/500px/api-documentation/blob/master/endpoints/photo/POST_photos.md)) with the photo details like title, description and so on. You may also want to set the `privacy` parameter to `1` if you wish to do a private upload or `0` for public upload.
```
Comment (mwstobo) [inline basics/upload.md]: I might add a note about the `auto_activate` parameter here too, as it's important to make sure that it's set.

Comment (mck182) [reply basics/upload.md]: @mwstobo done!

**Pull request merged**